### ISSUE TITLE
(TS) Eliminate already-listed cases in switch-case autocomplete dropdown

### ISSUE THREAD

**Issue by mk360** [microsoft/TypeScript#47873]
Search terms: switch, switch case, autocomplete, autocomplete switch case

Considering this example:

```ts
type MultiplePossibilities = 'a' | 'b' | 'c' | 'd';
let str: MultiplePossibilities;

// ...

switch (str) {
  case 
  break;
  case
}
```

When filling the first case, I receive a suggestion from the autocompletion with a selection ranging from `a` to `d`.  Which makes sense, since it's the first case and there's nothing before it. However, if I choose `a` in my first case and then move onto other cases, the autocompletion dropdown still suggests `a`.

The problem is that it makes me lose track of what cases need to be covered, especially since the IDE doesn't raise any warning in case (no pun intended) I added a case I already had covered. This can be especially bothersome in a sufficiently big [template literal type](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html).

## About my IDE

Version: 1.64.2
Date: February 9th, 2022
TS version: 4.5.5
OS: Windows NT_x64, v10.0.19043

**Comment by RyanCavanaugh** [comment]
Duplicate #13711
